tools/unix/generate_localizations.sh

```diff
@@ -1,110 +0,0 @@
-if [[ $TWINE == *".om/bin/twine" ]]; then
-  echo "Using the correctly patched submodule version of Twine"
-else
-  echo "Looks like you have a non-patched version of twine, try to uninstall it with '[sudo] gem uninstall twine'"
-  exit 1
-fi
-
-OMIM_DATA="$OMIM_PATH/data"
-STRINGS_PATH="$OMIM_DATA/strings"
-
-# Validate and format/sort strings files.
-STRINGS_UTILS="$OMIM_PATH/tools/python/strings_utils.py"
-"$STRINGS_UTILS" --validate --output
-"$STRINGS_UTILS" --types-strings --validate --output
-"$STRINGS_UTILS" "$STRINGS_PATH/sound.txt" --validate --output
-"$STRINGS_UTILS" "$OMIM_DATA/countries_names.txt" --validate --output
-"$STRINGS_UTILS" "$OMIM_PATH/iphone/plist.txt" --validate --output
-
-# Check for unused strings.
-CLEAN_STRINGS="$OMIM_PATH/tools/python/clean_strings_txt.py"
-"$CLEAN_STRINGS" --validate
-
-MERGED_FILE="$(mktemp)"
-cat "$STRINGS_PATH"/{strings,types_strings}.txt> "$MERGED_FILE"
-
-# TODO: Add validate-strings-file call to check for duplicates (and avoid Android build errors) when tags are properly set.
-"$TWINE" generate-all-localization-files --include translated --format android --untagged --tags android "$MERGED_FILE" "$OMIM_PATH/android/app/src/main/res/"
-"$TWINE" generate-all-localization-files --format apple --untagged --tags ios "$MERGED_FILE" "$OMIM_PATH/iphone/Maps/LocalizedStrings/"
-"$TWINE" generate-all-localization-files --format apple-plural --untagged --tags ios "$MERGED_FILE" "$OMIM_PATH/iphone/Maps/LocalizedStrings/"
-"$TWINE" generate-all-localization-files --format apple --file-name InfoPlist.strings "$OMIM_PATH/iphone/plist.txt" "$OMIM_PATH/iphone/Maps/LocalizedStrings/"
-"$TWINE" generate-all-localization-files --format jquery "$OMIM_DATA/countries_names.txt" "$OMIM_DATA/countries-strings/"
-"$TWINE" generate-all-localization-files --format jquery "$STRINGS_PATH/sound.txt" "$OMIM_DATA/sound-strings/"
-
-rm "$MERGED_FILE"
-
-# Generate list of languages and add list in gradle.properties to be used in build.gradle in resConfig
-SUPPORTED_LOCALIZATIONS="supportedLocalizations="$(sed -nEe "s/ +([a-zA-Z]{2}(-[a-zA-Z]{2,})?) = .*$/\1/p" "$STRINGS_PATH/strings.txt" | sort -u | tr '\n' ',' | sed -e 's/-/_/g' -e 's/,$//')
-# Chinese locales should correspond to Android codes.
-SUPPORTED_LOCALIZATIONS=${SUPPORTED_LOCALIZATIONS/zh_Hans/zh}
-SUPPORTED_LOCALIZATIONS=${SUPPORTED_LOCALIZATIONS/zh_Hant/zh_HK,zh_MO,zh_TW}
-SUPPORTED_LOCALIZATIONS=${SUPPORTED_LOCALIZATIONS/he/iw}
-SUPPORTED_LOCALIZATIONS=${SUPPORTED_LOCALIZATIONS/id/in}
-GRADLE_PROPERTIES="$OMIM_PATH/android/gradle.properties"
-if [ "$SUPPORTED_LOCALIZATIONS" != "$(grep supportedLocalizations "$GRADLE_PROPERTIES")" ]; then
-  sed -i.bak 's/supportedLocalizations.*/'"$SUPPORTED_LOCALIZATIONS"'/' "$GRADLE_PROPERTIES"
-  rm "$GRADLE_PROPERTIES.bak"
-fi
-
-# Generate locales_config.xml to allow users change app's language on Android 13+
-LOCALES_CONFIG="$OMIM_PATH/android/app/src/main/res/xml/locales_config.xml"
```

Comment:
and this file also

Reply:
Fix this PR https://github.com/organicmaps/organicmaps/pull/5665, resolves this case

Reply:
Let's keep the list static for now and merge #5665 later to avoid unnecessary ripple effects.